README.md

```diff
@@ -2,8 +2,8 @@
   <h1>Encryptic | The open source note-taking app</h1>
 
   <h3>🔖  &nbsp;🔏  &nbsp;📄&nbsp;</h3>
-  <p><strong>An encryption-focused open source note taking applciation.</strong><br>
-    Demo: https://encryptic.org
```

Comment:
As we are here, can you add a mention for the home page (https://encryptic.org) along with the demo (https://app.encryptic.org) ? :+1:

Reply:
Sure !